docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+.NET Products
```

Comment:
".NET Implementations" makes more sense to me than ".NET Products".  "Products" sounds like something a company has developed for sale, while implementations has a broader sense that seems to better suit the community-developed versions.